### PR TITLE
bug fix and test case for slot issue

### PR DESCRIPTION
**Pull request by NaveenGoli**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@NaveenGoli__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ngoli). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/Websites/Commands.Websites/Utilities/WebsitesClient.cs [hunk 1]
```diff
@@ -47,13 +47,20 @@ public WebSiteManagementClient WrappedWebsitesClient
 
         public WebSite CreateWebsite(string resourceGroupName, string webSiteName, string slotName, string location, string webHostingPlan)
         {
+            string webSiteSlotName = webSiteName;
+            if (string.IsNullOrEmpty(slotName) == false)
```
Comment (galcho) [inline src/ResourceManager/Websites/Commands.Websites/Utilities/WebsitesClient.cs]: you can use !string.IsNullOrEmpty(slotName)

Comment (NaveenGoli) [inline src/ResourceManager/Websites/Commands.Websites/Utilities/WebsitesClient.cs]: true, I think its stylistic fix?

**Comment by naveedaz** [comment]
Signed off from my perspective.

**Comment by markcowl** [comment]
@NaveenGoli reopend against the release branch here: #454

**Pull request closed without merging**